### PR TITLE
[ENG] Evaluate function to accept list of scorers

### PR DESCRIPTION
**Pull request by aiwalter**
#### What does this implement/fix? Explain your changes.
Function `from sktime.forecasting.model_evaluation import evaluate` to accept list of scorers, this is then added as new columns to the output data.

### CODE REVIEW

**Comment by RNKuhns** [comment]
@aiwalter this will be a nice incremental increase in functionality. I'll review once out of draft.

File: sktime/forecasting/model_evaluation/_functions.py [hunk 1]
```diff
@@ -8,13 +8,16 @@
...
 
 import numpy as np
+import numpy.typing as npt
 import pandas as pd
 
 from sktime.datatypes import check_is_scitype, convert_to
 from sktime.exceptions import FitFailedWarning
-from sktime.forecasting.base import ForecastingHorizon
+from sktime.forecasting.base import BaseForecaster, ForecastingHorizon
+from sktime.forecasting.model_selection._split import BaseSplitter
```
Comment (fkiraly) [inline sktime/forecasting/model_evaluation/_functions.py:20]: risk of circular imports, I would recommend to avoid doing this

**Review by fkiraly — CHANGES_REQUESTED** [review]
Makes sense.

The "returns" section in the docstring should document the change properly.

**Review by fkiraly — APPROVED** [review]
Thanks for completing the docstring.

**Pull request merged**